skins/oasis/css/modules/ContentReview.scss

```diff
@@ -48,8 +48,28 @@ $content-review-color-unsubmitted: #ffad00;
 
 .content-review-status-unsubmitted {
 	border-color: $content-review-color-unsubmitted;
+	animation: submitReminder 3s infinite;
 }
 
 .content-review-status-awaiting {
 	border-color: $content-review-color-awaiting;
 }
+
+/* Animation */
+@keyframes submitReminder {
```

Comment:
I want it to both start and end at 0 and wait for a while before the next iteration. Your solution makes it start from 0 and end at 10px everytime which looks like this: http://jsfiddle.net/kge01467/